src/pr-info.ts

```diff
@@ -430,12 +427,11 @@ function makeJsonCheckerFromCore(requiredForm: any, ignoredKeys: string[]) {
     };
 }
 
-function usersSayReadyToMerge(comments: PR_repository_pullRequest_comments_nodes[], users: string[], sinceDate: Date) {
-    return comments.some(comment =>
-        comment
-        && users.includes(comment.author?.login || " ")
+function usersSayReadyToMerge(comments: PR_repository_pullRequest_comments_nodes[], users: string[]) {
```

Comment:
The name should change though, since it's no longer a predicate.

Reply:
Done :heavy_check_mark: